src/main/java/uk/gov/hmcts/reform/pip/data/management/service/filegeneration/CrownPddaListFileConverter.java

```diff
@@ -48,10 +49,13 @@ public String convert(JsonNode artefact, Map<String, String> metadata, Map<Strin
     private void processDateInfo(Context context, JsonNode listNode, Language language) {
         JsonNode listHeader = listNode.get(LIST_HEADER);
         String publicationDateTime = listHeader.get("PublishedTime").asText();
-        context.setVariable("publicationDate",
-                            DateHelper.formatTimeStampToBst(publicationDateTime, language, false, false));
-        context.setVariable("publicationTime",
-                            DateHelper.formatTimeStampToBst(publicationDateTime, language, true, false));
+
+        boolean hasZ = publicationDateTime != null && publicationDateTime.toUpperCase(Locale.UK).endsWith("Z");
```

Comment:
Instead of appending the Z is it possible for the library to handle dates with / without the Z?